### DISCUSSION TITLE
How can I install it on Windows?

### DISCUSSION THREAD

**Discussion by Dhyfer1** [Schneegans/Burn-My-Windows#339] [Q&A] [answered]
As the title says. I have downloaded the zip file but can't find any executable. Can you give me the step by step instructions for Windows and put them in the readme file of the repository? 

Windows 11 Pro operating system. Dev channel. 64bits

**Comment by Schneegans** [comment, marked as answer]
Sorry, but Burn-My-Windows will never work on Windows. It is a GNOME Shell extension and as such it only works on Linux.

**Reply by Dhyfer1** [reply]
Damn. I thought it worked for Windows because of the program name "Burn my Windows". What a fool I was 🤦

Too bad it doesn't work on Windows. Do you know of any other alternative in this regard?

By the way, you can share the cursor seen in the repository video? I like it and I want to have it.
